tap_typeform/schemas/submitted_landings.json

```diff
@@ -34,6 +34,18 @@
     "hidden": {
       "type": ["null", "string"]
     },
+    "tags": {
+      "type": [
+        "null",
+        "array"
+      ],
+      "items": {
+        "type": [
+          "null",
+          "string"
+        ]
+      }
+    },
```

Comment:
I didn't realise it earlier but to keep pattern uniform, can we write schema on the same line?

Otherwise it looks good.